http-server/src/main/java/io/airlift/http/server/HttpServerConfig.java

```diff
@@ -70,6 +70,11 @@
     private List<String> includedCipherSuites = ImmutableList.of();
     private List<String> excludedCipherSuites = ImmutableList.of();
 
+    // For the SSL session-related config parameters
+    // a value of -1 means using the JVM default
+    private int sslSessionTimeout = -1;
```

Comment:
This should be a duration

Reply:
Oh also, lets have defaults.... 10k and 4 hours is most likely a fine place to start.